curvefs/proto/mds.proto

```diff
@@ -80,7 +86,7 @@ message FsInfo {
     required uint64 capacity = 5;
     required uint64 blockSize = 6;
     required uint32 mountNum = 7;
-    repeated string mountpoints = 8;
+    repeated Mountpoint mountpoints = 8;
```

Comment:
this will result in a compatible problem

Reply:
> this will result in a compatible problem

Remounting solves it